addon/components/frost-button.js

```diff
@@ -69,11 +74,22 @@ export default Component.extend(PropTypeMixin, {
     text: PropTypes.string,
     title: PropTypes.string,
     type: PropTypes.string,
-    vertical: PropTypes.bool
+    vertical: PropTypes.bool,
+
+    // state
+
+    // keywords
+    attributeBindings: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Ha - I was just going to type "this seems a bit boilerplate/redudant" then saw @job13er's last comment, yes - let's do that